src/nvim/buffer.c

```diff
@@ -2574,6 +2574,9 @@ static char_u *lasticon = NULL;
 
 void maketitle(void)
 {
+  // If we don't need to make the title, then don't.
+  if (!need_maketitle) return;
```

Comment:
Style guide says that `if` body must always be surrounded by `{}`: http://neovim.io/develop/style-guide.xml#Conditionals.